hack/images/merger/merger.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+SRC=${SRC:-"/yamls"}
+OUT=${OUT:-"/merged.yaml"}
+
+# prefix each file with its filename
+for filename in "${SRC}"/*.yaml; do
```

Comment:
WDYT about ignoring the extension? or maybe we should parametrize that? or simply support two most popular extensions, `yaml` and `yml`.

Reply:
let's ignore the extension